### DISCUSSION TITLE
I can't resolve a unittest GitHub Action error

### DISCUSSION THREAD

**Discussion by BECATRUE** [snu-quiqcl/qiwis#114] [General]
After I implemented a part of `swift.swift` unittest, an error occured.

It can be checked in Actions tab and the log is as below.
>qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, wayland-egl, wayland, wayland-xcomposite-egl, wayland-xcomposite-glx, webgl, xcb.

This seems to happen when there is even one function to test `Swift` class.
It does not happen when testing a function not in `Swift` class, e.g. `_get_argparser()` and `_read_setup_file()`.

I've been trying to solve it since yesterday, but I don't know the solution, so I'm looking for help.

**Comment by BECATRUE** [comment]
@kangz12345

**Reply by kangz12345** [reply]
What have you tried?
Does [this](https://cypsw.tistory.com/18) work?

**Reply by kangz12345** [reply]
Or [this](https://forum.qt.io/topic/127696/qt-qpa-plugin-could-not-load-the-qt-platform-plugin-xcb-in-even-though-it-was-found/20) could be helpful.

**Reply by BECATRUE** [reply]
> What have you tried? Does [this](https://cypsw.tistory.com/18) work?

Yes. I tried this, but it still failed.

**Comment by BECATRUE** [comment]
Wow, I did it to correct the error!

Referred to this stackoverflow [discussion](https://stackoverflow.com/questions/60692711/cant-create-python-qapplication-in-github-action), I changed it as below.
```console
pip install pyqt5
sudo apt-get install python3-pyqt5

xvfb-run `which coverage` run -m unittest discover tests
xvfb-run `which coverage` report -m
```

**Reply by BECATRUE** [reply]
In Ubuntu env, just `sudo apt-get install python3-pyqt5` is enough.
But, in GitHub Action, the following two statements are necessary.
